vm/resolutions.go

```diff
@@ -197,6 +182,13 @@ func (vm *VM) processAcceptedBlock(b *chain.StatelessBlock) {
 	if err := vm.SetLastProcessedHeight(b.Height()); err != nil {
 		vm.Fatal("failed to update the last processed height", zap.Error(err))
 	}
+
+	// Update subscriptions
+	for _, subscription := range vm.blockSubscriptions {
+		if err := subscription.Accept(b); err != nil {
+			vm.Fatal("subscription failed to process block", zap.Error(err))
+		}
+	}
```

Comment:
We need to send the subscription updates before setting the last processed height otherwise we will re-open this issue: https://github.com/ava-labs/hypersdk/issues/1145. We should guarantee to each subscription that they receive an accepted block notification at least once.